deploy/kubernetes/elastic-agent-managed-kubernetes.yaml

```diff
@@ -14,34 +14,40 @@ spec:
       labels:
         app: elastic-agent
     spec:
+      # To run Elastic Agent on Kubernetes master nodes
+      # Agents running on master nodes collect metrics from the control plane components (scheduler, controller manager) of Kubernetes
       tolerations:
         - key: node-role.kubernetes.io/master
           effect: NoSchedule
       serviceAccountName: elastic-agent
       hostNetwork: true
+      # 'hostPID: true' enables the Elastic Security integration to observe all process exec events on the host.
       # Sharing the host process ID namespace gives visibility of all processes running on the same host.
-      # This enables the Elastic Security integration to observe all process exec events on the host.
       hostPID: true
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: elastic-agent
           image: docker.elastic.co/beats/elastic-agent:8.3.0
           env:
+            # Set to 1 for enrollment into Fleet server. If not set, Elastic Agent is run in standalone mode
             - name: FLEET_ENROLL
               value: "1"
-            # Set to true in case of insecure or unverified HTTP
+            # Set to true to communicate with Fleet with either insecure HTTP or unverified HTTPS
             - name: FLEET_INSECURE
               value: "true"
-              # The ip:port pair of fleet server
+            # URL to enroll the Fleet Server into
             - name: FLEET_URL
               value: "https://fleet-server:8220"
-              # If left empty KIBANA_HOST, KIBANA_FLEET_USERNAME, KIBANA_FLEET_PASSWORD are needed
+            # The token to use for Fleet enrollment
```

Comment:
Dont think that line 41 is needed

Reply:
done https://github.com/elastic/elastic-agent/pull/708/commits/df21c4c24e4dfcb919b88da82eb5261e001e23bf

Reply:
Sorry seems missed that yesterday. I think we can add this link for the [token](https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html)

Reply:
Why not something like this ?
# Elasticsearch API key used to enroll Elastic Agents in Fleet (https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html#fleet-enrollment-tokens)

Reply:
@MichaelKatsoulis done https://github.com/elastic/elastic-agent/pull/708/commits/abec3438b1c808bc0bce892e998245f34fb368ad